Work on bugs/resource transfer2

demqn83 opened the pull request.

demqn83 committed:
Цвета перенес в ресурсы colors.xml.
Название картинок изменил, лишний файл png удалил.
Интервал между блоками изменен на 1 пустую строку.
Высоту у постера изменил.
Идентификаторы изменены на более подходящие.

demqn83 committed:
Цвета перенес в ресурсы colors.xml.
Название картинок изменил, лишний файл png удалил.
Интервал между блоками изменен на 1 пустую строку.
Высоту у постера изменил.
Идентификаторы изменены на более подходящие.

ivanlardis commented on app/src/main/res/layout/activity_movie_details.xml:
```diff
@@ -1,295 +1,290 @@
-        app:layout_constraintGuide_begin="16dp" />
+        <ImageView
+            android:id="@+id/poster_image_view"
+            android:layout_width="414dp"
```
![image](https://user-images.githubusercontent.com/11761610/100105334-48f68200-2e78-11eb-9986-68841328c5c0.png)
android:layout_width 0dp

ivanlardis commented on app/src/main/res/layout/activity_movie_details.xml:
```diff
@@ -1,295 +1,290 @@
-        app:layout_constraintTop_toBottomOf="@+id/textView20"
-        app:srcCompat="@drawable/ch" />
+        <ImageView
+            android:id="@+id/actor_photo_2_image_view"
```
https://mobikul.com/shapeableimageview-material-components/
со стилем
<item name="cornerFamily">rounded</item>
<item name="cornerSize">4dp</item>
чтоб обрезались углы


Просто сейчас вставлены картинки с обрезанными углами
Из-за этого выглядит как будто ок

demqn83 committed:
Изменена ширина у постера.
ImageView заменен на ShapeableImageView, и добавлен стиль со скручиванием углов.
